compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -1619,34 +1619,56 @@ pub type PlaceholderConst = Placeholder<BoundVar>;
 /// know their corresponding parameters. We (ab)use this by
 /// calling `type_of(param_did)` for these arguments.
 ///
+/// # Examples
+///
 /// ```rust
 /// #![feature(const_generics)]
 ///
 /// struct A;
 /// impl A {
-///     fn foo<const N: usize>(&self) -> usize { N }
+///     fn foo<const N: usize>(&self) -> [u8; N] { [0; N] }
+///     //           ^ const parameter
 /// }
 /// struct B;
 /// impl B {
-///     fn foo<const N: u8>(&self) -> usize { 42 }
+///     fn foo<const M: u8>(&self) -> usize { 42 }
 /// }
 ///
 /// fn main() {
 ///     let a = A;
-///     a.foo::<7>();
+///     let _b = a.foo::<7>();
+///     //               ^ const argument
 /// }
 /// ```
+///
+/// The important section here is the call `a.foo::<7>()`. We do not know which
```

Comment:
Why "section"? That sounds like a section in a book or so.
```suggestion
/// Consider the call `a.foo::<7>()`: we do not know which
```

Reply:
It still says "section", why did you resolve this comment?